### ISSUE TITLE
tf.estimator. adjust learning rate by loss on validation set

### ISSUE THREAD

**Issue by psy2013GitHub** [tensorflow/tensorflow#41740]
dear all, 

- by using tf.estimator, how to adjust learning rate by loss on validation set ?

> for example, if the loss in not decreasing, i want to half learning rate

thanks

**Comment by ravikyram** [comment]
@psy2013GitHub 

Please, go through [link1](https://stackoverflow.com/questions/45844320/how-to-use-a-decaying-learning-rate-with-an-estimator-in-tensorflow),[link2](https://stackoverflow.com/questions/61994141/loss-and-learning-rate-scaling-strategies-for-tensorflow-distributed-training-wh),[link3](https://github.com/ibab/tensorflow-wavenet/issues/267) and see if it helps you.

This question is better asked on StackOverflow since it is not a bug or feature request. There is also a larger community that reads questions there and provide better and faster support for such issues. Thanks!

**Comment by psy2013GitHub** [comment]
> @psy2013GitHub
> 
> Please, go through [link1](https://stackoverflow.com/questions/45844320/how-to-use-a-decaying-learning-rate-with-an-estimator-in-tensorflow),[link2](https://stackoverflow.com/questions/61994141/loss-and-learning-rate-scaling-strategies-for-tensorflow-distributed-training-wh),[link3](https://github.com/ibab/tensorflow-wavenet/issues/267) and see if it helps you.
> 
> This question is better asked on StackOverflow since it is not a bug or feature request. There is also a larger community that reads questions there and provide better and faster support for such issues. Thanks!

Thanks a lot

**Comment by ravikyram** [comment]
@psy2013GitHub 

Please feel free to close the issue if resolved.Thanks!
